subprojects/dependency-management/src/main/java/org/gradle/internal/component/external/model/DefaultMutableMavenModuleResolveMetadata.java

```diff
@@ -30,27 +33,40 @@
 import static org.gradle.internal.component.external.model.DefaultMavenModuleResolveMetadata.POM_PACKAGING;
 
 public class DefaultMutableMavenModuleResolveMetadata extends AbstractMutableModuleComponentResolveMetadata implements MutableMavenModuleResolveMetadata {
+    private final ImmutableAttributesFactory attributesFactory;
+    private final NamedObjectInstantiator objectInstantiator;
+    private final ExperimentalFeatures experimentalFeatures;
     private String packaging = "jar";
```

Comment:
This is really cosmetics, don't bother, but I like to separate `final` fields from mutable ones with an empty line.